src/viewport.js

```diff
@@ -1789,7 +1789,43 @@ $.Viewport.prototype = {
        */
       this.viewer.raiseEvent('flip', {flipped: state});
       return this;
-    }
+    },
+
+    /**
+     * Gets current max zoom pixel ratio
+     * @function
+     * @returns {Number} Max zoom pixel ratio
+     */
+    getMaxZoomPixelRatio: function() {
+        return this.maxZoomPixelRatio;
+    },
+
+    /**
+     * Sets max zoom pixel ratio
+     * @function
+     * @param {Number} ratio - Max zoom pixel ratio
+     * @param {Boolean} [constraints=false] - Apply constraints after setting ratio;
```

Comment:
This is a good flag to have, but I think the default for it should be `true`; that seems like the thing people would most likely want.

Also, maybe the flag should be called `applyConstraints` so it's more clear what it does (rather than people thinking that it is specifying which constraints or something).

Reply:
Done! Thanks for the feedback!